utilities/stats.rb

```diff
@@ -19,10 +19,10 @@ def initialize(stats = {})
     STATS.keys.each { |key| send "#{key}=", (stats[key] || 0) }
   end
 
-  def change_stats(other, sign)
+  def change_stats(_other, sign)
```

Comment:
see above

Reply:
See commit [4383558](https://github.com/narcisgiorgianchitescu/RPG_Game2.0/pull/83/commits/438355898d3dff84351d683e1ecefd27636500fa).

Reply:
I think the linter just didn't catch that the 'other' variable is actually expended before the eval call so it marked (and renamed it) as unused. I guess I'll just go back again to 'other' (so not _other).